.github/workflows/ci.yml

```diff
@@ -302,6 +307,7 @@ jobs:
           stage_key: property
 
   acceptanceTests:
```

Comment:
In the main ci.yml workflow the acceptance tests trigger similar to the docker build - only on `master` or `tag` but not on a PR so the secrets are safe